tests/Smoke/CiIntegrationTest.php

```diff
@@ -140,11 +140,6 @@ public function testIntegration(
             $steps[4],
         ]);
 
-        $optionalIncompatibilityWarning = 'PHP needs to be a minimum version of PHP 5.6.0 and maximum version of PHP 7.4.*.
```

Comment:
we should also update "max" here

Reply:
Why to keep it if never happens? I'd rather add PHP 8.1 build and then restore it with correct message.

Reply:
that's my point... we should not drop the test but ensure it's executing properly

Reply:
We are not dropping any test - this test still does it job. I'm only removing some dead code here, as clearly the "maximum version of PHP 7.4.*" part is not true for a long time.

I'm happy to add this back when we add build for PHP 8.1, but let's not add PHP 8.1 in PR titled "Remove references to PHP 5.6".

Reply:
I disagree.
we still have this in our codebase:
> php-cs-fixer:30:    fwrite(STDERR, "PHP needs to be a minimum version of PHP 7.1.3 and maximum version of PHP 8.0.*.\n");

instead of removing test checking this line, even if it's not fully executed on current CI config, update the test to match expected values (instead of REMOVING test update 5.6 to 7.1.3)